src/dataExplorer/components/TagSelector.tsx

```diff
@@ -115,13 +115,16 @@ const TagValues: FC<Prop> = ({loading, tagKey, tagValues}) => {
       <Accordion className="tag-selector-value">
         <div
           onClick={() => handleSelectTagKey(tagKey)}
-          data-testid="tag-selector--key"
+          data-testid="tag-selector-value--header-wrapper"
```

Comment:
def a better name, but the tests should key off of this div and not the `tag-selector-value` iirc

Reply:
@drdelambre i think the test does key off `tag-selector-value--header-wrapper` and not the `tag-selector-value`. is this what you mean?